File: app/src/main/java/com/djavid/bitcoinrate/model/dto/cryptowatch/Market.java
package com.djavid.bitcoinrate.model.dto.cryptowatch;


public class Market {

    private Integer id;
    private String exchange;
    private String pair;
    private Boolean active;
    private String route;


    public Integer getId() {
        return id;
    }
    public void setId(Integer id) {
        this.id = id;
    }

    public String getExchange() {
        return exchange;
    }
    public void setExchange(String exchange) {
        this.exchange = exchange;
    }

    public String getPair() {
        return pair;
    }
    public void setPair(String pair) {
        this.pair = pair;
    }

    public Boolean getActive() {
        return active;
    }
    public void setActive(Boolean active) {
        this.active = active;
    }

    public String getRoute() {
        return route;
    }
    public void setRoute(String route) {
        this.route = route;
    }

}

File: app/src/main/kotlin/com/djavid/bitcoinrate/model/cryptonator/Market.kt
package com.djavid.bitcoinrate.model.cryptonator


class Market {

    var market: String? = null
    var price: Double? = null
    var volume: Double? = null


    constructor()

    constructor(market: String, price: Double?, volume: Double?) : super() {
        this.market = market
        this.price = price
        this.volume = volume
    }

}
